src/webpack/webpack.config.js

```diff
@@ -0,0 +1,141 @@
+const Autoprefixer = require('autoprefixer');
+const PostCSSAssetsPlugin = require('postcss-assets-webpack-plugin');
+const PostCSSCustomProperties = require('postcss-custom-properties');
+const path = require('path');
+const rtl = require('postcss-rtl');
+const MiniCssExtractPlugin = require('mini-css-extract-plugin');
```

Comment:
We should log an issue to update [terra ui's recommended webpack config](https://github.com/cerner/terra-ui/blob/master/src/GettingStarted/ConfiguringWebpack.md#css-loaders-and-plugins) with these

Reply:
We should actually add documentation on the default config and the plugins etc used in the toolkit since we now provide it here and then terra-ui should reference that.

Reply:
Agreed, I was thinking about adding a readme to the webpack folder to explain the config.

Reply:
Issue Logged: https://github.com/cerner/terra-toolkit/issues/83